pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.nuxeo.ecm.distribution</groupId>
     <artifactId>nuxeo-distribution</artifactId>
-    <version>10.10-HF09-SNAPSHOT</version>
+    <version>10.10-HF10-SNAPSHOT</version>
```

Comment:
No more needed I guess

Reply:
10.10-HF10-SNAPSHOT is required, otherwise an error compilation occurs, because `WorkManagerRunWorkInFailure` will be missed. In other hand i will rebase the master into this branch, and in master we need 10.10 HF11.

Reply:
Yes, that's my point, once rebased you'll have the update to 10.10-HF11-SNAPSHOT, so this change is not needed in the current branch.

Reply:
exactly, it's what i did and now there is no modification on this pom.xml for changing the Nuxeo version.
thx